Package.swift

```diff
@@ -55,7 +55,10 @@ let package = Package(
         .target(
             /** Generic test support library */
             name: "TSCTestSupport",
-            dependencies: ["TSCBasic", "TSCUtility"]),
+            dependencies: ["TSCBasic", "TSCUtility"],
+            linkerSettings: [
+                .linkedFramework("XCTest", .when(platforms: [.iOS, .macOS, .tvOS, .watchOS])),
```

Comment:
Perhaps a naïve question, but why is this only for Darwin platforms?  I'm not that familiar with XCTest on Linux, but if this is a declaration that a non-test target needs to use XCTest, would that not apply to all platforms?

Reply:
That's a good question, maybe we should use `linkedLibrary` for other platforms?

Reply:
What was the resolution here?  Should we ask someone involved with XCTest?

Reply:
Currently not working on this, converted to draft for now.